tasks/libs/releasing/documentation.py

```diff
@@ -0,0 +1,194 @@
+import os
+from datetime import timedelta
+
+import requests
+
+from tasks.libs.owners.parsing import list_owners
+
+
+def create_release_page(version, freeze_date):
+    username = os.environ['ATLASSIAN_USERNAME']
+    password = os.environ['ATLASSIAN_PASSWORD']
+    auth = requests.auth.HTTPBasicAuth(username, password)
+    space_key = "agent"
+    parent_page_id = "2244936127"
+    # Make the POST request to create the page
+    domain = "https://datadoghq.atlassian.net/wiki"
+    url = f"{domain}/rest/api/content"
+    headers = {"Content-Type": "application/json"}
+    page_title = f"Agent {version}"
+    teams = get_releasing_teams()
```

Comment:
What do you think of sorting teams ? The generated page is pretty inconvenient to fill right now 😄 
```suggestion
    teams = sorted(get_releasing_teams())
```

Reply:
I think I agree!

Reply:
Maybe you can sort in the `get_releasing_teams` function